noted/accounts/v1/tchat.proto

```diff
@@ -0,0 +1,120 @@
+syntax = "proto3";
+
+package noted.accounts.v1;
+
+option go_package = "noted/accounts/v1";
+
+import "google/protobuf/timestamp.proto";
+
+// ConversationMessage is the representation of a tchat conversation stored
+// in a conversation context (in which conversation is it store, who write it,
+// content and creation date)
+message ConversationMessage {
+    string message_id = 1;
+    string conversation_id = 2;
+    string author_account_id = 3;
+    string content = 4;
+    google.protobuf.Timestamp created_at = 5;
+}
+
+// Conversation is the representation of a tchat conversation stored
+// in a group context (which group, with a title)
+message Conversation {
+    string conversation_id = 1;
+    string folder_id = 2;
+    string title = 3;
+}
+
+service TchatAPI {
+    rpc CreateConversation(CreateConversationRequest) returns (CreateConversationResponse) {}
+    rpc AddConversationMember(AddConversationMemberRequest) returns (AddConversationMemberResponse) {}
+    rpc RemoveConversationMember(RemoveConversationMemberRequest) returns (RemoveConversationMemberResponse) {}
+    rpc SendConversationMessage(SendConversationMessageRequest) returns (SendConversationMessageResponse) {}
+    rpc DeleteConversationMessage(DeleteConversationMessageRequest) returns (DeleteConversationMessageResponse) {}
+    rpc GetConversationMessage(GetConversationMessageRequest) returns (GetConversationMessageResponse) {}
+    rpc ListConversationMessage(ListConversationMessageRequest) returns (ListConversationMessageResponse) {}
+    rpc ListConversationMember(ListConversationMemberRequest) returns (ListConversationMemberResponse) {}
```

Comment:
Seems like you're missing a few endpoints (`ListConversations`, `UpdateConversationMessage`, etc). I've removed the members endpoints, fixed typos and added documentation.
```suggestion
    // Internal endpoint. Invoked upon group creation.
    rpc CreateConversation(CreateConversationRequest) returns (CreateConversationResponse) {}
    // Must be group member.
    rpc SendConversationMessage(SendConversationMessageRequest) returns (SendConversationMessageResponse) {}
    // Must be sender or group admin.
    rpc DeleteConversationMessage(DeleteConversationMessageRequest) returns (DeleteConversationMessageResponse) {}
    // Must be group member.
    rpc GetConversationMessage(GetConversationMessageRequest) returns (GetConversationMessageResponse) {}
    // Must be group member.
    rpc ListConversationMessages(ListConversationMessagesRequest) returns (ListConversationMessagesResponse) {}
```